main.c

```diff
@@ -126,7 +126,7 @@ open_by_handle_at (int mount_fd, struct file_handle *handle, int flags)
 
 #ifndef RENAME_EXCHANGE
 # define RENAME_EXCHANGE (1 << 1)
-# define RENAME_NOREPLACE (1 << 2)
+# define RENAME_NOREPLACE (1 << 0)
```

Comment:
Theoretically should RENAME_NOREPLACE be in its own block.  You could have a plaform that defines RENAME_EXCHANGE but not RENAME_NOREPLACE.

Reply:
yes that is possible in theory hence moved to its own block.